e2e/cypress/integration/mark_as_unread/mark_mentions_as_unread_spec.js

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {beUnread} from '../../support/assertions';
+
+import {markAsUnreadByPostIdFromMenu, verifyPostNextToNewMessageSeparator, switchToChannel} from './helpers';
```

Comment:
Thanks for the feedback. I have updated this. I am not sure why the build failed, it is getting a JavaScript Heap out of memory error but I don't think I changed anything that would cause this. Hoping the tests pass after pushing up this change.